lib/components/shopping-cart-widget-component.js

```diff
@@ -10,96 +10,41 @@ export default class ShoppingCartWidgetComponent extends BaseContainer {
 		super( driver, by.className( 'cart-toggle-button' ) );
```

Comment:
Can you update this to a css selector whilst your here? (It's an old selector added before we try to use CSS selectors exclusively)

Reply:
Done.